package.json

```diff
@@ -28,48 +28,46 @@
     "type": "git",
     "url": "git+https://github.com/openmrs/openmrs-esm-api.git"
   },
-  "keywords": [
-    "openmrs"
-  ],
-  "license": "MIT",
+  "license": "MPL-2.0",
   "bugs": {
     "url": "https://github.com/openmrs/openmrs-esm-api/issues"
   },
   "homepage": "https://github.com/openmrs/openmrs-esm-api#readme",
   "devDependencies": {
-    "@babel/core": "^7.5.5",
-    "@babel/preset-env": "^7.5.5",
-    "@babel/preset-react": "^7.0.0",
-    "@babel/preset-typescript": "^7.3.3",
-    "@types/jest": "^24.0.17",
+    "@babel/core": "^7.6.4",
+    "@babel/preset-env": "^7.6.3",
+    "@babel/preset-react": "^7.6.3",
+    "@babel/preset-typescript": "^7.6.0",
+    "@types/jest": "^24.0.19",
     "babel-eslint": "^11.0.0-beta.0",
-    "babel-jest": "^24.8.0",
+    "babel-jest": "^24.9.0",
     "babel-loader": "^8.0.6",
     "browserslist-config-openmrs": "^1.0.0",
     "clean-webpack-plugin": "^3.0.0",
     "concurrently": "^5.0.0",
-    "eslint": "^6.1.0",
-    "eslint-config-prettier": "^6.0.0",
+    "eslint": "^6.5.1",
+    "eslint-config-prettier": "^6.4.0",
     "eslint-config-ts-important-stuff": "^1.0.0",
-    "eslint-plugin-prettier": "^3.1.0",
-    "fhir.js": "0.0.20",
-    "fork-ts-checker-webpack-plugin": "^1.5.0",
-    "husky": "^3.0.3",
-    "jest": "^24.8.0",
-    "jest-cli": "^24.8.0",
+    "eslint-plugin-prettier": "^3.1.1",
+    "fhir.js": "0.0.22",
```

Comment:
0.0.22 introduced typescript defns, whereas 0.0.20 did not have them. The type defs there are really bad -- syntax errors. See related https://github.com/FHIR/fhir.js/pull/145 and https://github.com/FHIR/fhir.js/pull/143.

As a result, I went on a deep quest to figure out how to allow us to define our own types while also not relying on the types being in the node modules. I eventually found a hack that is shown below.